src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -561,8 +561,6 @@ OutTaskExecution(OUTFUNC_ARGS)
 	WRITE_UINT_FIELD(taskId);
 	WRITE_UINT_FIELD(nodeCount);
 
-	WRITE_ENUM_ARRAY(taskStatusArray, node->nodeCount);
-	WRITE_ENUM_ARRAY(transmitStatusArray, node->nodeCount);
 	WRITE_INT_ARRAY(connectionIdArray, node->nodeCount);
 	WRITE_INT_ARRAY(fileDescriptorArray, node->nodeCount);
```

Comment:
Is TaskExecution used at all after this PR?

Reply:
No it shouldn't be. Will remove it.

Reply:
0ae58ca